### DISCUSSION TITLE
The pink button crashes the test suite

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23519] [Ideas] [answered]
![Screenshot from 2023-05-12 20-53-30](https://github.com/ppy/osu/assets/101510334/55460d07-dd02-4156-8818-dc8551230128)

Edit: Here's more details, 
To reproduce:
1. `dotnet run --project osu.Game.Tests`
2. Find Online → AccountCreationOverlay → OverlayVisibility
3. On the tests, click "show manually"
4. Click "Let's create an account!"
5. Click on "I understand. This account isn't for me." or "support system". Clicking either one crashes the test suite.

**Comment by peppy** [comment, marked as answer]
There are two options you have here:

- Fix it and open a PR
- Don't click it.
